--- a/etc/init.d/minicoredumper.in	2016-11-22 10:44:28.060000000 +0000
+++ b/etc/init.d/minicoredumper.in	2016-11-22 10:49:24.656000000 +0000
@@ -33,13 +33,13 @@
 # Read configuration variable file if it is present
 [ -r @initdefaultsdir@/$NAME ] && . @initdefaultsdir@/$NAME
 
-# Load the VERBOSE setting and other rcS variables
-. /lib/init/vars.sh
+log_daemon_msg() {
+	echo "$@"
+}
 
-# Define LSB log_* functions.
-# Depend on lsb-base (>= 3.2-14) to ensure that this file is present
-# and status_of_proc is working.
-. /lib/lsb/init-functions
+log_end_msg() {
+	[ $1 -eq 0 ] || echo "$NAME: failed"
+}
 
 #
 # Function that starts the daemon/service
